NEWS.md

```diff
@@ -1,4 +1,5 @@
 ## formatters 0.5.1.9003
+ * `truncate_spans` function modified to handle matrix with a single column after pagination.
```

Comment:
truncate spans is a internal function, please modify the news entry to talk about the user-facing bug that was fixed instead.